scripts/operations.py

```diff
@@ -66,7 +66,7 @@ def sample(job):
 
 def auto(job):
     "This is a meta-operation to execute multiple operations."
-    from my_project import get_project
+    from my_project.project import get_project
```

Comment:
Ok, so I looked into this and the actual issue were the missing __init__.py files. Should be fixed now.